cmd/worker_cli/worker_cli.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/MinhNghiaD/jobworker/api/client"
+	"github.com/MinhNghiaD/jobworker/api/worker/proto"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/genproto/googleapis/rpc/errdetails"
+	"google.golang.org/grpc/status"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	address = kingpin.Flag("a", "server address").Default("127.0.0.1:7777").String()
+	// TODO add flags for certificate, private key and server CAs
+
+	// Start subcommand and its flags
+	start     = kingpin.Command("start", "Start a job on worker service.")
+	startCmd  = start.Flag("cmd", "command to be executed").Default("").String()
```

Comment:
I think with the empty string the RPC will more likely return with an error instead of running an actual job. I think it would be better if the user launches the request with their input instead of running a default command